pkg/controllers/dnspolicy/dns_helper.go

```diff
@@ -1,32 +1,27 @@
-package dns
+package dnspolicy
 
 import (
 	"context"
 	"errors"
 	"fmt"
-	"sort"
 	"strconv"
 	"strings"
 
+	"github.com/Kuadrant/multicluster-gateway-controller/pkg/_internal/slice"
+	"github.com/Kuadrant/multicluster-gateway-controller/pkg/apis/v1alpha1"
+	"github.com/Kuadrant/multicluster-gateway-controller/pkg/dns"
+	"github.com/Kuadrant/multicluster-gateway-controller/pkg/traffic"
```

Comment:
I'm not sure we have ever discussed a convention for grouping imprts, but generally  think we should keep all the pkg ones last and  in their own group. This is consistent with most other files.